src/OrchardCore.Modules/Orchard.Liquid/Filters/MetadataFilters.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using DotLiquid;
+using Microsoft.AspNetCore.Html;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.DependencyInjection;
+using Orchard.ContentManagement;
+using Orchard.Liquid.Drops;
+
+namespace Orchard.Liquid.Filters
+{
+    public static class MetadataFilters
+    {
+        public static string DisplayText(Context context, object input)
+        {
```

Comment:
So, with a `SqlQuery` here the input object is a `JTokenDrop` OR a `JObject` if i query document ids and check the `ReturnDocuments` option. Then it doesn't work, so, maybe we could try to convert this object to a `ContentItemDrop`.

But first, maybe good to have in both case (`ReturnDocuments` = false / true) the same type of `JTokenDrop`. I could do this by updating `SqlQuerySource` which is not there. In place of doing [this](https://github.com/OrchardCMS/Orchard2/blob/master/src/OrchardCore.Modules/Orchard.Queries/Sql/SqlQuerySource.cs#L64) if the option is true, i'm doing [this](https://github.com/OrchardCMS/Orchard2/blob/master/src/OrchardCore.Modules/Orchard.Queries/Sql/SqlQuerySource.cs#L76-L81) as when the option is false. So in both case we return a collection of `JObject`.

I say this because with `SqlQuery`, when using content item properties without `MetadataFilters`, to get it working in both cases, i needed to update `SqlQuerySource`.